components/butterfly/src/rumor/dat_file.rs

```diff
@@ -552,7 +556,7 @@ mod tests {
         // Now that the dat file content was written, re-read the content back in
         // to ensure underlying filesystem operations occurred successfully.
         let content = DatFileReader::read(dat_path).unwrap();
-        assert_eq!(content.header.version, 2);
-        assert_eq!(content.header.size, 64);
+        assert_eq!(content.header.version, CURRENT_HEADER_VERSION);
+        assert_eq!(content.header.size, HEADER_VERSION_2_SIZE);
```

Comment:
@jeremymv2 With the constant name changes, it might be more consistent to call this `CURRENT_HEADER_VERSION_SIZE`.